client/src/pages/events/Form.js

```diff
@@ -0,0 +1,601 @@
+import { gql } from "@apollo/client"
+import API from "api"
+import AdvancedMultiSelect from "components/advancedSelectWidget/AdvancedMultiSelect"
+import {
+  EventSeriesOverlayRow,
+  LocationOverlayRow,
+  OrganizationOverlayRow
+} from "components/advancedSelectWidget/AdvancedSelectOverlayRow"
+import AdvancedSingleSelect from "components/advancedSelectWidget/AdvancedSingleSelect"
+import AppContext from "components/AppContext"
+import CustomDateInput from "components/CustomDateInput"
+import DictionaryField from "components/DictionaryField"
+import * as FieldHelper from "components/FieldHelper"
+import Fieldset from "components/Fieldset"
+import Messages from "components/Messages"
+import Model from "components/Model"
+import NavigationWarning from "components/NavigationWarning"
+import NoPaginationTaskTable from "components/NoPaginationTaskTable"
+import {
+  jumpToTop,
+  mapPageDispatchersToProps,
+  PageDispatchersPropType,
+  useBoilerplate
+} from "components/Page"
+import RichTextEditor from "components/RichTextEditor"
+import { FastField, Field, Form, Formik } from "formik"
+import _isEmpty from "lodash/isEmpty"
+import _isEqual from "lodash/isEqual"
+import { Event, EventSeries, Location, Organization, Task } from "models"
+import CreateNewLocation from "pages/locations/CreateNewLocation"
+import pluralize from "pluralize"
+import PropTypes from "prop-types"
+import React, { useContext, useState } from "react"
+import { Button, FormSelect } from "react-bootstrap"
+import { connect } from "react-redux"
+import { useNavigate } from "react-router-dom"
+import LOCATIONS_ICON from "resources/locations.png"
+import ORGANIZATIONS_ICON from "resources/organizations.png"
+import TASKS_ICON from "resources/tasks.png"
+import { RECURSE_STRATEGY } from "searchUtils"
+import Settings from "settings"
+import utils from "utils"
+import { TaskOverlayRow } from "../../components/advancedSelectWidget/AdvancedSelectOverlayRow"
```

Comment:
Please import this from `"components/advancedSelectWidget/AdvancedSelectOverlayRow"`.